testing/ruby-ttytest/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: Eivind Uggedal <eu@eju.no>
+pkgname=ruby-ttytest
+_gemname=ttytest
+pkgver=0.5.0
+pkgrel=0
+pkgdesc="Acceptance test framework for interactive console applications"
+url="https://github.com/jhawthorn/ttytest"
+arch="noarch"
+license="MIT"
+depends="ruby tmux"
+checkdepends="ruby-minitest ruby-rake ruby-bundler"
+source="$pkgname-$pkgver.tar.gz::$url/archive/v$pkgver.tar.gz
+	skip-yard.patch
+	fix-flaky-test.patch
+	gemspec.patch"
+builddir="$srcdir/$_gemname-$pkgver"
+
+build() {
+	cd "$builddir"
```

Comment:
no need to `cd "$builddir"` this is done implicitly

Reply:
Thanks. The perils of not contributing for 4 years...